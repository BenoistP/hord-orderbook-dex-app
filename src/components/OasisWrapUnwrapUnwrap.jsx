import React, { PureComponent } from 'react';
import { PropTypes } from 'prop-types';
// import ImmutablePropTypes from 'react-immutable-proptypes';

import styles from './OasisWrapUnwrapUnwrap.scss';

const propTypes = PropTypes && {};
const defaultProps = {};

class OasisWrapUnwrapUnwrap extends PureComponent {
  render() {
    return (
      <div className={styles.base}>
      </div>
    );
  }
}

OasisWrapUnwrapUnwrap.displayName = 'OasisWrapUnwrapUnwrap';
OasisWrapUnwrapUnwrap.propTypes = propTypes;
OasisWrapUnwrapUnwrap.defaultProps = defaultProps;
export default OasisWrapUnwrapUnwrap;
