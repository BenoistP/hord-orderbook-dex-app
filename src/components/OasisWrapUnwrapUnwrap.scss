.balance {
  //margin-top: 4em;
}

.widgetFrame {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
}