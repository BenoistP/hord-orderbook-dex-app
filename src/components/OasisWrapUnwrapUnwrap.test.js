/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';
import { shallow } from 'enzyme';
import OasisWrapUnwrapUnwrap from './OasisWrapUnwrapUnwrap';

describe('(Component) OasisWrapUnwrapUnwrap', () => {
  it('should render', () => {
    const props = {
      children: (
        <span>test</span>
      ),
    };
    const wrapper = shallow(
      <OasisWrapUnwrapUnwrap {...props}/>,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
