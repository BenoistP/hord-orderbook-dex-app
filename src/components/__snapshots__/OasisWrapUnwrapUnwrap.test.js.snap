// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) OasisWrapUnwrapUnwrap should render 1`] = `<div />`;
